Storage/SQL/CommandSyncTable.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  CommandSyncTable.swift
+//  Client
+//
+//  Created by Emily Toop on 6/29/15.
+//  Copyright (c) 2015 Mozilla. All rights reserved.
+//
+
+import Shared
+
+class CommandSyncTable<T>: GenericTable<SyncCommand> {
+    override var name: String { return "commands" }
+    override var version: Int { return 1 }
+
+    override var rows: String { return join(",", [
+        "guid TEXT PRIMARY KEY",
+        "client_guid TEXT REFERENCES clients(guid)",
+        "url TEXT",
+        "title TEXT",
+        "action TEXT NOT NULL",
+        "last_used INTEGER",
+        ])
+    }
+
+
+    override func getInsertAndArgs(inout item: SyncCommand) -> (String, [AnyObject?])? {
+        var args = [AnyObject?]()
```

Comment:
We have `Args` as a typealias here. And prefer `let` when you know the contents in advance.